pkg/helmexec/exec.go

```diff
@@ -599,3 +601,21 @@ func resolveOciChart(ociChart string) (ociChartURL, ociChartTag string) {
 	ociChartURL = fmt.Sprintf("oci://%s", ociChart[:urlTagIndex])
 	return ociChartURL, ociChartTag
 }
+
+func (helm *execer) ShowChart(chartPath string) (chart.Metadata, error) {
```

Comment:
Please add a unit test for this ShowChart. @felipecrs Thanks very much.

Reply:
@yxxhero done.